systemtest/src/test/java/io/strimzi/systemtest/LogLevelST.java

```diff
@@ -0,0 +1,223 @@
+    };
+
+    private static final Map<String, String> OPERATORS_LOGGERS = new HashMap<String, String>() {
+        {
+            put("rootLogger.level", DEBUG);
+        }
+    };
+
+    private static final Map<String, String> MIRROR_MAKER_LOGGERS = new HashMap<String, String>() {
+        {
+            put("mirrormaker.root.logger", TRACE);
+        }
+    };
+
+    private static Resources classResources;
+
+
+    @Test
+    void testKafkaLoggers() {
+        int duration = TimeMeasuringSystem.getCurrentDuration(testClass, testClass, operationID);
+        assertTrue(checkLogLevel(KAFKA_LOGGERS, duration, KAFKA_MAP), "Kafka's log level is set properly");
+    }
+
+    @Test
+    void testZookeeperLoggers() {
+        int duration = TimeMeasuringSystem.getCurrentDuration(testClass, testClass, operationID);
+        assertTrue(checkLogLevel(ZOOKEEPER_LOGGERS, duration, ZOOKEEPER_MAP), "Zookeeper's log level is set properly");
+    }
+
+    @Test
+    void testTOLoggers() {
+        int duration = TimeMeasuringSystem.getCurrentDuration(testClass, testClass, operationID);
+        assertTrue(checkLogLevel(OPERATORS_LOGGERS, duration, TO_MAP), "Topic operator's log level is set properly");
+    }
+
+    @Test
+    void testUOLoggers() {
+        int duration = TimeMeasuringSystem.getCurrentDuration(testClass, testClass, operationID);
+        assertTrue(checkLogLevel(OPERATORS_LOGGERS, duration, UO_MAP), "User operator's log level is set properly");
+    }
+
+    @Test
+    void testKafkaConnectLoggers() {
+        int duration = TimeMeasuringSystem.getCurrentDuration(testClass, testClass, operationID);
+        assertTrue(checkLogLevel(CONNECT_LOGGERS, duration, CONNECT_MAP), "Kafka connect's log level is set properly");
+    }
+
+    @Test
+    void testMirrorMakerLoggers() {
+        int duration = TimeMeasuringSystem.getCurrentDuration(testClass, testClass, operationID);
+        assertTrue(checkLogLevel(MIRROR_MAKER_LOGGERS, duration, MM_MAP), "Mirror maker's log level is set properly");
+    }
+
+    private boolean checkLogLevel(Map<String, String> loggers, int since, String configMapName) {
+        boolean result = false;
+        for (Map.Entry<String, String> entry : loggers.entrySet()) {
+            LOGGER.info("Check log level setting since {} seconds. Logger: {} Expected: {}", since, entry.getKey(), entry.getValue());
+            String configMap = kubeClient.get("configMap", configMapName);
+            String loggerConfig = String.format("%s=%s", entry.getKey(), entry.getValue());
+            result = configMap.contains(loggerConfig);
+
+            if (result) {
+                String log = kubeClient.searchInLog(STATEFUL_SET, kafkaClusterName(CLUSTER_NAME), since, ERROR);
+                result = log.isEmpty();
+            }
+        }
+
+        return result;
+    }
+
+    @BeforeEach
+    void logTest(TestInfo testInfo) {
+        LOGGER.info("Running {} in namespace {}", NAMESPACE, testInfo.getDisplayName());
```

Comment:
Method arguments in wrong ordering. You need to swap NAMESPACE and testInfo.getDisplayName().